src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/Items.js

```diff
@@ -10,11 +12,60 @@ type Props = {
     skin?: Skin,
 };
 
-export default class Items extends React.PureComponent<Props> {
+@observer
+export default class Items extends React.Component<Props> {
+    @observable expandedWidth = 0;
+    @observable showText = true;
+
     static defaultProps = {
         skin: 'light',
     };
 
+    parentRef: { current: null | HTMLDivElement } = React.createRef();
+    childRef: { current: null | HTMLUListElement } = React.createRef();
```

Comment:
I was just trying because i wanted to know why it was failing back then so i came up with this. In this case, i knew that i only need to know the 'current' property of the ref, and this could be a HTMLElement or null. It works without any problems.